packages/eslint-plugin-okta/babel.config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "presets": [[
+      "@babel/preset-env", {
+          "useBuiltIns": "usage",
+          "loose": true,
+          "modules": "auto",
+          "corejs": 3
```

Comment:
**NON-BLOCKING**

We're probably using corejs, but I'd personally recommend against it. Once we get rid of IE11 support, this would be the first thing I'd recommend removing.

It looks like we're bringing in all of corejs v3. Pretty sure we're supposed to be selective about it. Maybe it tree-shakes properly? I remember at Minted, that not being the case. It was something like 30% of the bundle size.